main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+var c = &Config{
+	Modules: map[string]Module{
+		"http_2xx": Module{
+			Prober:  "http",
+			Timeout: time.Duration(10 * time.Second),
+		},
+	},
+}
+
+func TestPrometheusTimeout(t *testing.T) {
+
+	sc.C = c
```

Comment:
Changing globals in tests is best avoided where possible. Pass in what you need to the function.